.github/workflows/plugins.yml

```diff
@@ -80,30 +80,72 @@ jobs:
         run: yarn test:plugins:ci
       - if: env.MAJOR_VERSION == '3'
         uses: codecov/codecov-action@v2
-  aerospike:
+  aerospike-4:
     runs-on: ubuntu-latest
     services:
       aerospike:
-        image: aerospike:ce-6.4.0.3
+        image: aerospike:ce-5.7.0.15
         ports: 
           - "127.0.0.1:3000-3002:3000-3002"
     env:
       PLUGINS: aerospike
       SERVICES: aerospike
-      PACKAGE_VERSION_RANGE: '4.0.0 - 5.7.0'
+      PACKAGE_VERSION_RANGE: '4.0.0 - 5.4.0'
     steps:
       - uses: actions/checkout@v2
       - uses: ./.github/actions/testagent/start
       - uses: ./.github/actions/node/setup
       - run: yarn install --ignore-engines
       - uses: ./.github/actions/node/oldest
       - run: yarn test:plugins:ci
-      - run: echo "PACKAGE_VERSION_RANGE=>=5.8.0" >> "$GITHUB_ENV"
-      - uses: ./.github/actions/node/20 # currently the latest version of aerospike only supports node 20
-      - run: yarn test:plugins:ci
       - if: always()
         uses: ./.github/actions/testagent/logs
       - uses: codecov/codecov-action@v2
+  aerospike-5:
+    runs-on: ubuntu-latest
+    services:
+      aerospike:
+        image: aerospike:ce-6.4.0.3
+        ports: 
+          - "127.0.0.1:3000-3002:3000-3002"
+    env:
+      PLUGINS: aerospike
+      SERVICES: aerospike
+      PACKAGE_VERSION_RANGE: '5.5.0 - 5.7.0'
```

Comment:
Why is the max range up to 5.7? We should continue testing future versions automatically, otherwise it could break without us noticing.

Reply:
Later on in the steps i run: run: echo "PACKAGE_VERSION_RANGE=>=5.8.0" >> "$GITHUB_ENV". This should automatically test the newest versions automatically on node 20

Reply:
The reason that is done is because '5.5.0 - 5.7.0' support node 16. Where as >=5.8.0 only support node >= 18.